### PR TITLE
Feature/activity timeline

### PR DESCRIPTION
**Pull request by sherlock270**
# Description

Created activity timeline that shows only messages currently. Also changed the structure of Message Board to accommodate the timeline tab

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change status
- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Tested locally

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: frontend/src/MessageBoard/components/MessageBoard.js [hunk 1]
```diff
@@ -187,8 +184,8 @@ class MessageBoard extends React.Component {
 						<AddMessage
 							closeHandler={this.closeModalHandler}
 							stopProp={this.stopProp}
-							team={this.props.match.params.team}
-							user={this.state.user}
+							team={this.state.team}
```
Comment (koennecker) [inline frontend/src/MessageBoard/components/MessageBoard.js]: Should be this.props.team, right?

File: frontend/src/MessageBoard/components/UserList.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ export default class UserList extends Component {
 				>
 					Close
 				</button>
-				<Query query={query.FIND_TEAM} variables={{ id: team }}>
+				<Query query={query.FIND_TEAM} variables={{ id: team._id }}>
```
Comment (koennecker) [inline frontend/src/MessageBoard/components/UserList.js]: should just be {id: team}

File: frontend/src/MessageBoard/components/MessageBoard.js (hunk 1 above)
Comment (sherlock270) [reply frontend/src/MessageBoard/components/MessageBoard.js]: Fixed

File: frontend/src/MessageBoard/containers/MessageBoardContainer.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+			<MsgContainer>
+				<button name="messageboard" onClick={this.tabChange}>
+					Message Board
+				</button>
+				<button name="timeline" onClick={this.tabChange}>
+					Activity Timeline
+				</button>
+				{this.state.messageboard ? (
+					<MessageBoard
+						user={this.props.currentUser}
```
[the remark this answers is not in the document]
Comment (sherlock270) [reply frontend/src/MessageBoard/containers/MessageBoardContainer.js]: Fixed

File: frontend/src/MessageBoard/components/MessageBoard.js (hunk 1 above)
Comment (koennecker) [reply frontend/src/MessageBoard/components/MessageBoard.js]: It's still using `this.state.team`

**Pull request merged**